src/components/__tests__/VesselDashboard.spec.js

```diff
@@ -59,7 +58,7 @@ describe('VesselDashboard.vue', () => {
 
   test('renders GoogleMap component', () => {
     const { wrapper } = setup()
-    expect(wrapper.findAll(GoogleMap)).toHaveLength(1)
+    expect(wrapper.findAll(VesselDashboardGoogleMap)).toHaveLength(1)
```

Comment:
I have been using `wrapper.contains(Component).toBe(true)` when I wanted to check if a component exists inside a given component. Yours looks more strict though. 👍